spec/features/user_adds_vendor_spec.rb

```diff
@@ -15,8 +17,18 @@
     member = FactoryGirl.create(:user)
```

Comment:
This should actually be two different tests - one test to make sure an un-signed-in user doesn't see the button, and one test to make sure the signed in user can add a vendor.